Src/BootCamp.Chapter/Logger.cs

```diff
@@ -6,12 +6,12 @@ namespace BootCamp.Chapter
 {
     public class Logger
```

Comment:
I guess that could work, but then this would be a specialized logger, like ``MainMenuLogger``.